installers/msi-language/ActiveState/Tracker.cs

```diff
@@ -0,0 +1,70 @@
+﻿using GAPixel;
+using GoogleAnalyticsTracker.Core;
+using GoogleAnalyticsTracker.Core.TrackerParameters;
+using GoogleAnalyticsTracker.Simple;
+using Microsoft.Deployment.WindowsInstaller;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+
+namespace ActiveState
+{
+    public sealed class TrackerSingleton
+	{
+        private static readonly Lazy<TrackerSingleton> lazy = new Lazy<TrackerSingleton>(() => new TrackerSingleton());
+        private static string GoogleAnalyticsUserAgent = "UA-118120158-2";
+
+        private readonly SimpleTracker _tracker;
+        private readonly string _cid;
+
+        public static TrackerSingleton Instance {  get { return lazy.Value; } }
+
+        public TrackerSingleton()
+		{
+            var simpleTrackerEnvironment = new SimpleTrackerEnvironment(Environment.OSVersion.Platform.ToString(),
+                Environment.OSVersion.Version.ToString(),
+                Environment.OSVersion.VersionString);
+            this._tracker = new SimpleTracker(GoogleAnalyticsUserAgent, simpleTrackerEnvironment);
+            this._cid = GetInfo.GetUniqueId();
+  		}
+
+        public async Task<TrackingResult> TrackEventAsync(string category, string action, string label, long value = 1)
+        {
+            var eventTrackingParameters = new EventTracking
+            {
+                Category = category,
+                Action = action,
+                Label = label,
+                Value = value,
+            };
+
+            eventTrackingParameters.ClientId = this._cid;
+
+            return await this._tracker.TrackAsync(eventTrackingParameters);
```

Comment:
Nit: why use async if you're going to await anyway?

Reply:
The short answer: `TrackAsync` is what the C# library provides.  

The long answer: Every async function is supposed to have an `await` in C#.  The way I understand the idea behind it: You can write your control flow, as if you were writing a synchronized control flow, but things are actually running in threads and in parallel with nicer functionality to handle issues that occur with concurrency, like cancellation and callbacks.  I think every time you hit an `await` statement, the thread that you are executing in, could also change...